bokehjs/src/lib/models/glyphs/scatter.ts

```diff
@@ -22,6 +23,17 @@ export class ScatterView extends MarkerView {
     return MultiMarkerGL
   }
 
+  override _set_data(indices: number[] | null): void {
```

Comment:
This should be moved to `MarkerView` class in `models/glyphs/marker.ts`, as it's where `size` property is defined and where it should be handleded.